oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/query/AsyncProfilerTaskLog.java

```diff
@@ -31,8 +31,8 @@
 @AllArgsConstructor
 @Builder
 public class AsyncProfilerTaskLog {
+    // task id
     private String id;
```

Comment:
Yes, this is a semantic bug I discovered recently. Originally, graphql `queryAsyncProfilerTaskProgress` interface returned the `id` of log instead of `taskId`, but it is defined as `taskId` in graphql comment. Here, it is modified to taskId to conform to the semantics of graphql